test/unit/whitehall/publishing_api_test.rb

```diff
@@ -346,4 +346,27 @@ class Whitehall::PublishingApiTest < ActiveSupport::TestCase
     PublishingApiUnpublishingWorker.expects(:perform_async).with(1)
     Whitehall::PublishingApi.unpublish_async(unpublishing)
   end
+
+  test ".publish handles specific exceptions" do
+    raises_exception = lambda { |_,_,_|
+      response = MiniTest::Mock.new
+      body = '{"error":{"code":422,"message":"base path=/world/organisations/uk-science-innovation-network-in-belgium conflicts with content_id=9bb528b1-743c-4ea2-a323-5b1aaf41818b and locale=en","fields":{"base":["base path=/world/organisations/uk-science-innovation-network-in-belgium conflicts with content_id=9bb528b1-743c-4ea2-a323-5b1aaf41818b and locale=en"]}}}'
```

Comment:
Is it possible to set this up as a normal Rails hash (over multiple lines), then convert to JSON using `to_json`?  It might make it slightly easier to read.